terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.test.ts

```diff
@@ -669,13 +669,15 @@ runner_types:
       instance_type: c5.2xlarge
       os: linux
       min_available: 1
+      max_available_enforced: false
```

Comment:
Doesn't it seem a bit clunky to have a `max_available_enforced` value?  Can't even guess if it defaults to true or false without digging deep into the code?

Instead, what about treating any runner_type that doesn't have `max_available` defined has having no max limit requested?

Reply:
after chatting directly, I agree with you. It would be better to have this more clearly defined

Reply:
Since 0 or less is not really a valid configuration for `max_available` maybe we can have the check be if the value is 0 or less.